tests/main/auto-refresh-retry/task.yaml

```diff
@@ -0,0 +1,75 @@
+summary: Check that auto-refresh is retried after a permanent network error
+
+# FIXME: core18 right now does not have a canonical model. This means
```

Comment:
It is worth checking if this is till the case. Perhaps this comment is stale here and in some other tests?

Reply:
core18 enabled and works with this test, comment removed.